tests/integration-tests/tests/performance_tests/test_openfoam.py

```diff
@@ -0,0 +1,82 @@
+import json
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
+
+# timeout in seconds
+OPENFOAM_INSTALLATION_TIMEOUT = 1800
+OPENFOAM_JOB_TIMEOUT = 600
```

Comment:
The OpenFOAM job takes more than 10 minutes because the first time it run it's not only execute the job but also build and install many things. I would keep this to 90 minutes and I suggest to add a comments saying why it'ìs so high.